hadoop-ozone/cli-shell/src/main/java/org/apache/hadoop/ozone/shell/snapshot/ListSnapshotDiffHandler.java

```diff
@@ -37,38 +38,32 @@ public class ListSnapshotDiffHandler extends Handler {
   @CommandLine.Mixin
   private BucketUri snapshotPath;
 
-  @CommandLine.Option(names = {"-s", "--status"},
+  @CommandLine.Option(names = {"-js", "--job-status"},
```

Comment:
Please avoid multi-char short options (`-js`).

- They should be combinable (`-j -s` should be the same as `-js`)
- Not every long option needs a short one
- Command-line completion is coming (#8030)

Reply:
Removed `-js` option.